plugins/rest-plugin/src/test/java/ru/sbtqa/tag/api/entries/fromresponse/FromResponseSecondEndpointEntry.java

```diff
@@ -15,16 +15,16 @@
     @FromResponse(path = "email")
     private String emailFromPreviousRequest;
 
-    @FromResponse(path = "email", endpointEntry = FromResponseFirstEndpointEntry.class)
+    @FromResponse(path = "email", endpoint = FromResponseFirstEndpointEntry.class)
     private String emailFromSpecifiedRequest;
 
-    @FromResponse(endpointEntry = FromResponseFirstEndpointEntry.class, header = Default.HEADER_PARAMETER_NAME_1)
+    @FromResponse(endpoint = FromResponseFirstEndpointEntry.class, header = Default.HEADER_PARAMETER_NAME_1)
     private String firstHeaderValue;
 
-    @FromResponse(endpointEntry = FromResponseFirstEndpointEntry.class, path = "nonexistent", necessity = false)
+    @FromResponse(endpoint = FromResponseFirstEndpointEntry.class, path = "nonexistent", necessity = false)
```

Comment:
It will be good to rename necessity value to optional and set default value to false.